Owncloud iOs Client/InstantUpload/InstantUpload.h

```diff
@@ -0,0 +1,32 @@
+//
+//  InstantUpload.h
+//  Owncloud iOs Client
+//
```

Comment:
You have to use the header that we use on the other files. Check any other class and copy it.
Of course you can put your name as "created" 😉